### PR TITLE
API model docs

### PR DESCRIPTION
**Pull request by jtweed**
### What is this PR trying to achieve?

Put in place initial expanded API docs.

Note that another pull request will follow with the final text. This is just so we can see what's in the ontology docs as they stand.

You can see the output of this here:

https://wellcomecollection-catalogue.api-docs.io/v0/works/works

### Who is this change for?

Developers using our Swagger docs.

### CODE REVIEW

File: api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayWork.scala [hunk 1]
```diff
@@ -53,9 +59,14 @@ case object DisplayWork {
   }
 }
 
-case class DisplayIdentifier(identifierScheme: String, value: String) {
-  @JsonProperty("type")
-  val ontologyType: String = "Identifier"
+@ApiModel(
```
Comment (kenoir) [inline api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayWork.scala]: Love this. Documentation in code as documentation!

File: api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayResultList.scala [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.sksamuel.elastic4s.http.search.SearchResponse
+import io.swagger.annotations.{ApiModel, ApiModelProperty}
+
+@ApiModel(
+  value="ResultList",
+  description="A list of things."
+)
+case class DisplayResultList(
```
Comment (kenoir) [inline api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayResultList.scala]: Nice change.

**Comment by kenoir** [comment]
Travis is upset because you need to run `make format` to ensure the Scala code is formatted in a way that makes our build happy.

**Comment by kenoir** [comment]
Also as this changes the API output slightly is causes some of the API tests to fail. The tests will need updating to match the expected output.

File: api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayResultList.scala (hunk 2 above)
Comment (jtweed) [inline api/src/main/scala/uk/ac/wellcome/platform/api/models/DisplayResultList.scala]: A nice change that didn't entirely work! Fixing now.

**Pull request merged**